src/makefile.all

```diff
@@ -653,24 +672,30 @@ $(LIBARG): $(__MAKEFILES__)
 @ifdef WIN32
 $(LINKARG): $(__MAKEFILES__)
 	%create $^@
-	@%append $^@ option implib=$(IMP_LIB), quiet, map, verbose,
-	@%append $^@ caseexact, map=$(WATT_DLL:.dll=.map), res=$(RESOURCE)
+	@%append $^@ option quiet, verbose
+	@%append $^@ debug all
+	@%append $^@ opt map=$(WATT_DLL:.dll=.map)
+	@%append $^@ opt implib=$(IMP_LIB), res=$(RESOURCE)
+	@%append $^@ reference __DLLstart_
 	@for %f in ($(OBJS)) do @%append $^@ file %f
 @endif
 
 clean: .SYMBOLIC
-	- @del $(OBJPATH)*.obj
-	- @del $(OBJPATH)cflags.h
-	- @del $(OBJPATH)cflagsbf.h
-	- @del $(TARGETS)
-	- @del $(C_ARGS)
-	- @del $(LIBARG)
-	- @del $(PKT_STUB)
+	@if exist $(OBJPATH)*.obj $(RM) $(OBJPATH)*.obj
```

Comment:
Where is `RM` set? And doesn't all `wmake` programs support `rm -f` as an internal command?
If so, why this `@if exit xx` test? It just adds more noise to the makefiles.

Reply:
RM is defined on begining of watcom section with others macros as host OS dependent macro.
"if exist" is simple check for development that you see what is realy deleted and you need not ignore return code for del/rm, that if some problem with deletion you get a message.
If you don't want it simply replace it by -@$(RM)  ...

Reply:
Anyway rm command is internal wmake command from OW 2.0. For compatibility with OW 1.9 it uses external rm command on Linux only, because OW 1.9 wmake doesn't have internal rm command.